cwl/src/test/scala/cwl/ThreeStepExample.scala

```diff
@@ -1,123 +0,0 @@
-package cwl
```

Comment:
this is not a test, just an example that was a maintenance burden as well as an inaccurate 3step representation.